website/static/js/registrationUtils.js

```diff
@@ -463,37 +463,42 @@ RegistrationEditor.prototype.create = function(schemaData) {
 };
 RegistrationEditor.prototype.submit = function() {
     var self = this;
-    
+
```

Comment:
In order for builds to pass on Travis, they must pass cleanly through jshint. 

This code has multiple stylistic errors. Please install and run jshint and fix the breakages.